src/applications/travel-pay/components/submit-flow/pages/IntroductionPage.jsx

```diff
@@ -22,82 +23,75 @@ const IntroductionPage = ({ onStart }) => {
+            </p>
+            <va-link
+              external
+              href={`${TRAVEL_PAY_INFO_LINK}#eligibility-for-general-health`}
+              text="Travel reimbursement eligibility"
+            />
+          </va-process-list-item>
+          <va-process-list-item header="File your claim">
+            <p>
+              If you’re only claiming mileage, you can file online right now.
+              We’ll just ask you a few questions—you won’t need receipts.
+            </p>
+            {data &&
+              !data.travelPayClaim?.claim &&
+              !data.isOutOfBounds &&
+              data.isPast && (
+                <va-link-action
+                  onClick={e => onStart(e)}
+                  href="javascript0:void"
+                  text="File a mileage-only claim"
+                />
+              )}
+            <p>
+              If you’re claiming other expenses, like lodging, meals, or tolls,
+              you will need receipts for these expenses. You can file online
+              through the Beneficiary Travel Self Service System (BTSSS), by
+              mail, or in person.
+            </p>
+            <va-link
+              external
+              href={TRAVEL_PAY_INFO_LINK}
+              text="Learn how to file claims for other expenses"
+            />
+          </va-process-list-item>
+        </va-process-list>
+        <va-alert status="info" visible>
+          <h3 id="set-up-direct-deposit" slot="headline">
+            Set up direct deposit
+          </h3>
+          <p className="vads-u-margin-y--0">
+            You have to set up direct deposit to receive travel reimbursement.
+            If you’ve already done this, no additional steps are needed.
           </p>
           <va-link
             external
-            href={TRAVEL_PAY_INFO_LINK}
-            text="Learn how to file claims for other expenses"
+            href="/resources/how-to-set-up-direct-deposit-for-va-travel-pay-reimbursement/"
+            text="Set up direct deposit"
           />
-        </va-process-list-item>
-      </va-process-list>
-      <va-alert status="info" visible>
-        <h3 id="set-up-direct-deposit" slot="headline">
-          Set up direct deposit
-        </h3>
-        <p className="vads-u-margin-y--0">
-          You have to set up direct deposit to receive travel reimbursement. If
-          you’ve already done this, no additional steps are needed.
-        </p>
-        <va-link
-          external
-          href="/resources/how-to-set-up-direct-deposit-for-va-travel-pay-reimbursement/"
-          text="Set up direct deposit"
-        />
-      </va-alert>
-      <div
-        className="omb-info--container vads-u-margin-y--3"
-        style={{ paddingLeft: '0px' }}
-      >
-        <va-omb-info
-          res-burden={10}
-          omb-number="2900-0798"
-          exp-date="11/30/2027"
-        />
-      </div>
+        </va-alert>
+      </DowntimeWindowAlert>
     </div>
```

Comment:
This should remain here. The OMB info was a must-have.

Reply:
Merge issue, good catch!